### ISSUE TITLE
Allow ignoring missing @params

### ISSUE THREAD

**Issue by phiresky** [gajus/eslint-plugin-jsdoc#1169]
## Motivation

Often many parameters are self-explanatory, but some aren't. It would be great if `jsdoc/check-param-names` had an option to skip the missing 

## Current behavior

```ts
/**
 * @param additionalChecks If true, it is checked if the argument is within the bounds 0<x<100
 */
function add10(target: number, additionalChecks: boolean) {
```

Error:
Expected @param names to be "target, additionalChecks". Got "additionalChecks".

## Desired behavior

Some parameters are self-explanatory. Adding a comment about them just increases noise and reduces readability. I understand if the default should stay as is but a config option like` "jsdoc/check-param-names":["warn",{ disableMissingParamChecks: true}]` would be great.

## Alternatives considered

The only option I saw is disabling the rule completely - but that would also remove the checks for whether the param names are correct which I like.

**Comment by seanpoulter** [comment]
Would you welcome a PR? If yes, is there any order to the tests in **test/rules/assertions/checkParamNames.js** or should I append to the end? Thanks folks!

**Comment by brettz9** [comment]
Yes, a PR which adds a new option would be welcome.

The testing order is fairly arbitrary, though it will be published in the docs in order. You're welcome to just add to the end of the invalid and valid tests since it would be a new option.
